tests/datasets/test_mapinwild.py

```diff
@@ -98,20 +98,20 @@ def test_invalid_split(self) -> None:
 
     def test_not_downloaded(self, tmp_path: Path) -> None:
         with pytest.raises(RuntimeError, match="Dataset not found"):
-            MapInWild(root=str(tmp_path))
+            MapInWild(root=tmp_path)
 
     def test_downloaded_not_extracted(self, tmp_path: Path) -> None:
         pathname = os.path.join("tests", "data", "mapinwild", "*", "*")
         pathname_glob = glob.glob(pathname)
-        root = str(tmp_path)
+        root = tmp_path
         for zipfile in pathname_glob:
             shutil.copy(zipfile, root)
-        MapInWild(root, download=False)
+        MapInWild(str(root), download=False)
```

Comment:
Why str?

Reply:
This was probably added earlier to fix mypy error when ``MapInWild`` did not support ``pathlib.path``. Have fixed this and gone through other test files to fix similar issues.